### PR TITLE
Add Persian Support

### PR DESCRIPTION
**Pull request by AlirezaIvaz**
Hi. I added Persian language support and applied Custom font for Persian language page.

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -97,6 +103,7 @@
                     <option value="fr">Français</option>
                     <option value="ko">한국어</option>
                     <option value="it">Italiano</option>
+					<option value="fa">فارسی</option>
```
Comment (RhinosF1) [inline index.php]: Should this be indented so far?

**Comment by AlirezaIvaz** [comment]
Unfortunately I didn't get what you meant!

**Comment by RhinosF1** [comment]
> Unfortunately I didn't get what you meant!

Apologies, the merge displayed funny. Ignore me.

**Comment by AlirezaIvaz** [comment]
> > Unfortunately I didn't get what you meant!
> 
> Apologies, the merge displayed funny. Ignore me.

Where is it? Can you explain a bit?

**Comment by RhinosF1** [comment]
> > Apologies, the merge displayed funny. Ignore me.
> 
> Where is it? Can you explain a bit?

Click your latest commit and then the yellow box

**Comment by AlirezaIvaz** [comment]
> Click your latest commit and then the yellow box

It is very interesting! I have not even touched the English text!

File: index.php [hunk 2]
```diff
@@ -25,9 +25,8 @@
 <?php
 require_once __DIR__ . '/translations.php';
 ?>
-
 <!DOCTYPE html>
-<html>
+<html lang="<?php if(isset($_GET['lang'])) { echo $_GET['lang']; } else { echo 'en'; } ?>" dir="<?php if(isset($_GET['lang'])) { echo 'rtl'; } else { echo 'ltr'; } ?>">
```
Comment (paladox) [inline index.php:29]: Looking at this, if lang is not set it will use right to left, english is the default language?

Comment (paladox) [reply index.php:29]: You probably want to do it so only certain languages use the rtl feature.

File: index.php [hunk 3]
```diff
@@ -41,6 +40,13 @@
 
     <link rel="stylesheet" href="css/all.min.css">
     <link rel="stylesheet" href="css/miraheze.min.css">
+	<?php
```
Comment (paladox) [inline index.php:43]: Code style (space between brackets).

**Comment by paladox** [comment]
Also need rebase.

File: index.php (hunk 2 above)
Comment (AlirezaIvaz) [reply index.php:29]: Oh, I didn't pay attention and I left the code incomplete, I'll correct it soon. I just wanted to change the direction when it was Persian. I'm so sorry.

**Comment by Universal-Omega** [comment]
Has not been updated since January; also severely out-of-date..

**Pull request closed without merging**